components/tools/OmeroWeb/omeroweb/settings.py

```diff
@@ -785,11 +793,32 @@ def report_settings(module):
 LANGUAGE_CODE = 'en-gb'
 
 # SECRET_KEY: A secret key for this particular Django installation. Used to
-# provide a seed in secret-key hashing algorithms. Set this to a random string
-# -- the longer, the better. django-admin.py startproject creates one
-# automatically.
-# Make this unique, and don't share it with anybody.
-SECRET_KEY = '@@k%g#7=%4b6ib7yr1tloma&g0s2nni6ljf!m0h&x9c712c7yj'
+# provide a seed in secret-key hashing algorithms. Set this to a random string,
+# the longer, the better. Make this unique, and don't share it with anybody.
+try:
+    SECRET_KEY
+except NameError:
+    secret_path = os.path.join(OMERO_HOME, 'var',
```

Comment:
@manics thanks, web logs already depends on ``OMERO_HOME``. One the time comes for https://trello.com/c/KRlpnQnW/344-remove-uses-of-omero-home this will be rewritten anyway.